titus-client/src/main/java/com/netflix/titus/runtime/connector/jobmanager/CachedReadOnlyJobOperations.java

```diff
@@ -141,4 +143,9 @@ public CachedReadOnlyJobOperations(JobDataReplicator replicator) {
                     return false;
                 });
     }
+
+    @Override
+    public Optional<List<ContainerState>> getEphemeralTaskStatus(String taskId) {
```

Comment:
As I understand we have two cases here:
1. if task exists, return empty list
2. if task does not exit, return Optional.empty()